include/application/tasks/ventilation.h

```diff
@@ -9,11 +9,20 @@
 
 #define MAX_CIRCUIT_PRESSURE_FOR_OPENING_INS_VALVE_CSP 32 //todo replace this value by the real value
 
+#include <stdint.h>
 /*
  * SLS-0055
  */
 void ventilation();
 
+/*
+ * Gets the respiratory rate inputted by clinician
+ * Step 1: Get clinician selected inspitory rate (SRS-0001) OR default inspiratory rate (SRS-0004)
+ * Step 2: Calculate duration of a breath cycle in milliseconds
+ * Step 3: Assign new inspiratory rate to breath_cycle_duration
+ */
+uint32_t calculate_breath_cycle_duration();

```

Comment:
optional: 

I know you didn't choose this name... (I am even sure that I did).. but it sounds not really english... maybe compute instead of calculate would be better? what do you think ?

Reply:
I personally think it's fine actually, it just sounds like it's a verb taken out from a complete sentence to be used here. I'll leave it if no one has any conflicting opinions.